variables.tf

```diff
@@ -1,11 +1,11 @@
 variable "environment" {}
 variable "domain" {}
-variable "subnet_ids" {}
+variable "private_subnet_ids" {}
 variable "vpc_id" {}
 variable "ami_id" {}
 variable "instance_type" {}
 variable "ingress_cidr_blocks" {}
 variable "zone_id" {}
 variable "email" {}
 variable "dns" {}
-
+variable "size" {}
```

Comment:
A more descriptive name for this variable should be `volume_size` (also update references)

Reply:
Done, changed to volume_size